lib/core.js

```diff
@@ -5,6 +5,7 @@ import pathModule from "path"
 
 import cleanUp from "./utils/clean_up"
 import { getFolderName, makeComponentName, createFiles, prepareFiles } from "./utils/files"
```

Comment:
if you create a `hookFiles`, then it could be interesting to rename `files` to `baseFiles` or something like than

Reply:
`utils/files` needs to be asbtracted into two separate files - one that contains the component's logic, and one that contains generic file operations. But this can be done in another issue. What do you think ?

Reply:
works for me